components/alone/listIntro/ListIntroLanding.tsx

```diff
@@ -133,15 +135,22 @@ function ListIntroLanding() {
   };
 
   const handleFolderNameChange = (e: React.ChangeEvent<HTMLInputElement>) => {
-    e.stopPropagation();
     setFolderName(e.target.value);
   };
 
   const handleListNameChange = (e: React.ChangeEvent<HTMLInputElement>) => {
-    e.stopPropagation();
     setListName(e.target.value);
   };
 
+  const handleKeyboardFocus = () => {
+    setIsKeyboardFocused(true);
+    setIsMobile(/iPhone|iPad|iPod|Android/i.test(navigator.userAgent));
+  };
+
+  const handleKeyboardBlur = () => {
+    setIsKeyboardFocused(false);
+  };
+
```

Comment:
input 창에 focus가 가서 키보드가 활성화되면 `isKeyboardFocused`를 true로 설정하고, blur되면 false로 설정합니다.

모바일 기기로 접속했는지 확인하는 방법은, userAgent에 특정 키워드가 포함되어있는지 아닌지 여부를 기준으로 판별합니다.
userAgent : 브라우저가 서버에게 보내는 HTTP 요청 헤더 중 하나로, 해당 브라우저에 대한 정보를 담고 있음

Reply:
`useIntallGuide.tsx` 에 `isIos` 함수가 있는데
`isAndroid`, `isMobile` 유틸 함수를 따로 추가하고 유틸 파일로 분리하여 사용하면 어떨까요?
```suggestion
  const handleKeyboardFocus = () => {
    setIsKeyboardFocused(true);
    setIsMobile(/iPhone|iPad|iPod|Android/i.test(navigator.userAgent));
  };

  const handleKeyboardBlur = () => {
    setIsKeyboardFocused(false);
  };

```